.github/release.yml

```diff
@@ -6,6 +6,13 @@ changelog:
       exclude:
         labels:
           - dependencies
-    - title: Dependencies 📦
+          - feature toggled
+          - technical changes only
+    - title: "Experimental features for testing 😎"
+        - feature toggled
+    - title: "Technical changes 🔧"
```

Comment:
Also inspired by JB, I like this icon:
```suggestion
    - title: "Technical changes 🛠"
```

Reply:
Okay, I managed to insert the coloured version. There's not much in it, I think. :shrug:

Reply:
Both versions were coloured for me, I guess they appear differently in different environments (perhaps something to consider more). This is mine:
![Screen Shot 2023-08-23 at 3 14 04 pm](https://github.com/openfoodfoundation/openfoodnetwork/assets/4188088/0c15582e-e3db-473e-bde7-d2d574d95928)